### PR TITLE
[9.0][MIG] crm_helpdesk

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migrated from Odoo, 8.0 version.

### CODE REVIEW

File: crm_helpdesk/data/crm_helpdesk_data.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<openerp>
+    <data noupdate="1">
+    <!--
+    Case Sections
+    -->
+    </data>
```
Comment (LoisRForgeFlow) [inline crm_helpdesk/data/crm_helpdesk_data.xml]: Why this file?

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot :+1:

File: crm_helpdesk/data/crm_helpdesk_data.xml (hunk 1 above)
Comment (MiquelRForgeFlow) [reply crm_helpdesk/data/crm_helpdesk_data.xml]: This file existed in v8 😂

Comment (LoisRForgeFlow) [reply crm_helpdesk/data/crm_helpdesk_data.xml]: I see... a pity to remove it, the nostalgia is very powerful :laughing: I would do it though!

Comment (MiquelRForgeFlow) [reply crm_helpdesk/data/crm_helpdesk_data.xml]: now it's removed

**Pull request closed without merging**